### PR TITLE
added timezone to timeRelative patterns

### PR DESCRIPTION
**Pull request by lorem--ipsum**

### CODE REVIEW

File: src/filter-pattern/pattern-time-relative.ts [hunk 1]
```diff
@@ -134,28 +138,28 @@ export const TIME_RELATIVE_PATTERN_DEFINITION: FilterPatternDefinition<TimeRelat
...
         shiftDuration,
         shiftStep,
+        timezone,
       };
     },
     isValid(_pattern): boolean {
       return true;
     },
-    toExpression(pattern): SqlExpression {
+    toExpression(pattern, timezone?: string): SqlExpression {
```
Comment (vogievetsky) [inline src/filter-pattern/pattern-time-relative.ts:147]: surely timezone should come from the pattern

Comment (lorem--ipsum) [reply src/filter-pattern/pattern-time-relative.ts:147]: oh of course, duh 😅

File: src/filter-pattern/pattern-time-relative.ts [hunk 2]
```diff
@@ -82,6 +84,8 @@ export const TIME_RELATIVE_PATTERN_DEFINITION: FilterPatternDefinition<TimeRelat
       const rangeStep = -timeShift.getArgAsNumber(2)!;
       if (!(rangeStep > 0)) return;
 
+      const timezone = timeShift.getArgAsString(3);
```
Comment (vogievetsky) [inline src/filter-pattern/pattern-time-relative.ts:87]: Please add checks to make sure that, if a timezone is used it is the same in all the instances of the pattern

**Review by vogievetsky — APPROVED** [review]
gr 🎱

**Pull request merged**